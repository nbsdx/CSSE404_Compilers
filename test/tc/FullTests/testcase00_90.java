class NAME
{
	public static void main(String[] name) {
	}
}
class name
{
	int x;
	boolean y;
	public int getx() {
		return x;
	}
	public boolean gety() {
		return y;
	}
	public boolean compare()
	{
		return null||true;
	}
}
