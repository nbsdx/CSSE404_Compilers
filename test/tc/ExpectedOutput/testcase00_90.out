Left argument of type null does not match expected type boolean for operator OR
